vms/registration/tests/test_functional_admin.py

```diff
@@ -173,15 +173,16 @@ def test_name_fields(self):
         # test length of first-name, last-name not exceed 30
         page.get_admin_registration_page()
 
-        entry = ['admin-username','admin-password!@#$%^&*()_','admin-first-name-!@#$%^&*()_','admin-last-name-!@#$%^&*()_','admin-email1@systers.org','admin-address','admin-city','admin-state','admin-country','9999999999','admin-org']
+        entry = ['admin-username','admin-password!@#$%^&*()_','admin-first-name-!@#$%^&*()_lolwatneedlength','admin-last-name-!@#$%^&*()_lolwatneedlength','admin-email1@systers.org','admin-address','admin-city','admin-state','admin-country','9999999999','admin-org']
```

Comment:
@mayank-agarwal-96 As the default length was changed to 30, the test would fail. Because `admin-first-name-!@#$%^&*()_` only has 28 characters. So, I changed it to `admin-first-name-!@#$%^&*()_lolwatneedlength` which has 44 characters. So that the test would succeed.